nominal/nominal.py

```diff
@@ -48,11 +48,23 @@ def set_base_url(base_url: str) -> None:
     For staging environments: "https://api-staging.gov.nominal.io/api".
     For local development: "https://api.nominal.test".
     """
-    _config.get_token(base_url)
+    _config.get_token(base_url) # @vtupuri - why is this needed? error checking?
```

Comment:
aside: generally more common to use the format `# TODO(username): comment` for this

Reply:
good question. IIRC the intention is generally error checking, like making sure you do _have_ a token for the URL before you use it? is kinda gross tho